action.yml

```diff
@@ -1,22 +1,40 @@
-name: 'Composite Run Steps Action Template'
+name: 'Setup K6 Performance Testing'
 
-description: 'Add description here'
+description: 'Downloads and installs k6 at the specified version and configures OS with k6 optimized settings for stress testing.'
 
 inputs:
-  input-1:
-    description: 'Add a description here'
+  version:
+    description: The version of k6 to install
     required: true
-
-outputs:
-  output-1:
-    description: 'Add description here'
-    value: '${{ steps.step_1.outputs.world }}'
+  architecture:
+    description: 'Target operating system architecture for K6 to use. Examples: x86, x64. Will use system architecture by default.'
+    required: false
+    default: 'amd64'
 
 runs:
   using: 'composite'
   steps:
-    - name: Step 1
-      id: step_1
+    - name: 'Download K6'
+      id: download
+      shell: bash
+      working-directory: '${{ github.action_path }}'
+      env:
+        VERSION: ${{ inputs.version }}
+        ARCH: ${{ inputs.architecture }}
+      run: |
+        node ./dist/index.js
+    - name: Tune OS
+      id: tune_os
```

Comment:
Just as an FYI for this step and the one above it, unless you are using it in an output or subsequent step, I think the convention is generally to not provide the ID.